packages/meditrak-app/app/sync/syncMigrations.js

```diff
@@ -38,6 +38,10 @@ const migrations = {
   7: async (synchroniser, setProgressMessage) => {
     await resyncRecordTypes(synchroniser, setProgressMessage, ['survey_screen_component']);
   },
+  // Resync all entities so that the new attributes column come through
+  8: async (synchroniser, setProgressMessage) => {
+    await resyncRecordTypes(synchroniser, setProgressMessage, ['entity']);
+  },
```

Comment:
@edmofro  yesterday when I was testing the upgrade with Sepi I found out that none of the attributes column in Entity were synced :(. `Attributes` data in Entity is new so I think resyncing all of the entities is the only way. What do you reckon?

Reply:
Damn damn damn, this is like 14000 records. I guess you're right though! What a shame, sucks for those people on low speed connections in the Pacific. Still, great catch guys.